docs/usage.rst

```diff
@@ -25,69 +25,60 @@ We do not claim compatibility to the following RFCs. They might work though.
 - :rfc:`9073` - Event Publishing Extensions to iCalendar
 - :rfc:`9253` - Support for iCalendar Relationships
 
-File structure
+iCalendar File structure
 --------------
 
-An iCalendar file is a text file (utf-8) with a special format. Basically it
-consists of content lines.
+An iCalendar file is a text file (utf-8) with a special format.
 
-Each content line defines a property that has 3 parts (name, parameters,
-values). Parameters are optional.
+It consists of **content lines**,
+with each content line defining a property that has 3 parts: name, parameters and value. Parameters are optional.
 
-A simple content line with only name and value could look like this::
+Example 1: a simple content line (with only name and value)::
 
   BEGIN:VCALENDAR
```

Comment:
For each example, the default lexer is Python by virtue of the preceding line's `::`, but should be text. Would you please update each example with this pattern, replacing `::` with `.`, inserting the `code-block` directive, and indenting by four spaces the code example?

```suggestion
.. code-block:: text

    BEGIN:VCALENDAR
```

Reply:
Done!